docs/user-guide/cross-sections.ipynb

```diff
@@ -7,273 +7,180 @@
    "source": [
```

Comment:
<p>"UXarray allows for vertical cross‑sections to be performed along arbitrary great‑circle arcs (GCAs) and lines of constant latitude or longitude."</p><p> </p><p>How about rephrasing the first paragraph as above? "Temporal cross-section" might be misleading as we are doing only vertical cross-sections as part of this functionality, and temporal is more of a subject of slicing I think.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>